tx/rawdb.go

```diff
@@ -198,13 +198,18 @@ func ReadNumConfirmedRawTxs(db ethdb.Reader, addr common.Address) uint64 {
 	return n
 }
 
-func WriteNumConfirmedRawTxs(db ethdb.KeyValueWriter, addr common.Address, n uint64) {
+func WriteNumConfirmedRawTxs(db ethdb.KeyValueWriter, quit chan struct{}, addr common.Address, n uint64) {
```

Comment:
I think adding `wg` to `rcm` can remove using `quit` channel here, cause db will not closed when `Write*` function is called

Reply:
Oh my fault. yeah. There is no need checking `quit` chan in `WriteNumConfirmedRawTxs` function. Will fix it.

Reply:
I forgot to commit this https://github.com/Onther-Tech/plasma-evm/pull/115/commits/a1dab595693bf81601316a6dd8da7d1bd5bf16c4